### PR TITLE
[IMP] accounting: improving snailmail documentation

### PR DESCRIPTION
**Pull request by Donapi**
[ID: 3099154](https://www.odoo.com/web#id=3099154&cids=1&model=project.task&view_type=form)
**Do not forward to 15 and 16:** how to access the Paper format must be updated first

### CODE REVIEW

File: content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst [hunk 1]
```diff
@@ -4,42 +4,51 @@
...
 
 .. important::
-   Make sure to have your customer’s address set correctly, and that it includes a country,
-   before sending the letter.
+   Your document must respect the following rules to pass the validation before being sent:
+
+   - Margins must be 5 millimeters on all sides. As Odoo forces the outer margins by filling them
+     with white before sending the snailmail, it can results in the user's custom being cut off if
+     it protrudes into the margins. To check the margins, go to :menuselection:`General Settings
+     --> Technical --> Reporting section: Paper format`.
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst:35]: I couldn't find "technical" and the "reporting section"; here's the path I took to find the paper format in v14 
```suggestion
     it protrudes into the margins. To check the margins, go to :menuselection:`Settings --> General
     Settings`, scroll down to the :guilabel:`Business Documents` section, and open the external
     link.
```

Also, I couldn't find it in Odoo 15 and Odoo 16 --> do you know if there's a way to check the margins in these versions? If not, it should be removed for the two versions

File: content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst [hunk 2]
```diff
@@ -4,42 +4,51 @@
...
-| *Snailmail* is an *In-App Purchase (IAP)* service which requires prepaid stamps (=credits)
-  to work. Sending one document consumes one stamp.
-| To buy stamps, go to :menuselection:`Accounting --> Configuration --> Settings --> Snailmail`
-  and click on *Buy credits*, or go to :menuselection:`Settings --> Odoo IAP` and click on
-  *View my Services*.
+Snailmail is an :doc:`../../../../general/in_app_purchase` service that requires prepaid stamps
+(=credits) to work. Sending one document consumes one stamp.
+
+To buy stamps, go to :menuselection:`Accounting --> Configuration --> Settings --> Snailmail` and
+click on :guilabel:`Buy credits`, or go to :menuselection:`Settings --> Odoo IAP` and click on
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst:50]: if you include the section earlier, it should be included here as well (section is: In-App Purchases)

File: content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst [hunk 3]
```diff
@@ -4,42 +4,51 @@
...
 
-| *Snailmail* is an *In-App Purchase (IAP)* service which requires prepaid stamps (=credits)
-  to work. Sending one document consumes one stamp.
-| To buy stamps, go to :menuselection:`Accounting --> Configuration --> Settings --> Snailmail`
-  and click on *Buy credits*, or go to :menuselection:`Settings --> Odoo IAP` and click on
-  *View my Services*.
+Snailmail is an :doc:`../../../../general/in_app_purchase` service that requires prepaid stamps
+(=credits) to work. Sending one document consumes one stamp.
+
+To buy stamps, go to :menuselection:`Accounting --> Configuration --> Settings --> Snailmail` and
```
Comment (LoredanaLrpz) [inline content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst:49]: if you include the section earlier, it should be included here as well (section is: Customer invoices)

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Coucou, je suis dispo aujourd'hui pour qu'on check mes commentaires ensemble =)

File: content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst [hunk 4]
```diff
@@ -4,42 +4,51 @@
...
-Set up Snailmail
-================
+Snailmail configuration
+=======================
 
-| Go to :menuselection:`Accounting --> Settings --> Snailmail` and activate the feature.
-| If you would like this option to be a default feature, enable *Send by Post* under
-  *Default Sending Options*.
+Go to :menuselection:`Accounting --> Configuration --> Settings --> Customer invoices section` to
+activate Snailmail.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst:14]: In this case, as I am referring to the feature in general, is the guilabel mandatory? I would think it is optional. I added it anyway :)

File: content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst (hunk 1 above)
Comment (Donapi) [reply content/applications/finance/accounting/receivables/customer_invoices/snailmail.rst:35]: Actually, "Technical" is available in Debug mode, but I understood we avoid to advise users to activate it. I updated as per your suggestion. For V15 and V16, I will investigate to see what are the options.

**Review by LoredanaLrpz — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Comment by Donapi** [comment]
@fw-bot ignore

**Comment by Donapi** [comment]
@fw-bot up to master

**Pull request closed without merging**